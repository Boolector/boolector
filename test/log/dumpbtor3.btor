1 var 8
2 var 8
3 var 1
4 root 8 1
5 root 8 2
6 root 1 3
