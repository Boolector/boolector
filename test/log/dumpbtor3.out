1 var 8
2 zero 8
3 eq 1 1 2
4 root 1 -3
5 var 8
6 eq 1 5 2
7 root 1 -6
